docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md

```diff
@@ -0,0 +1,93 @@
+# Implementing Cascading Settings for Tool Inputs
```

Comment:
make the doc file name consistent with the doc title?  how-to-use-cascading-inputs?

Reply:
make the doc file name consistent with the doc title